### PR TITLE
Basic Popup

### PR DESCRIPTION
**Pull request by RyanMarshall5765**
Adds basic popup on marker click. Displays all information from the GeoJSON in a table.

**Issues / To be worked on** 
1. ~The commas from the GeoJSON (in between object commas) are pulled in and put at the top of the table.~(.map() returns an array. Uses .toString() to convert to a string making them .join(',') fixed by changing the join to .join('') the empty string.)
2. ~Any value that is an array (Alternate place names etc.) the key is returned as the index of that value instead of the name of the key.~ (Fixed using Joe's recommendation)

### CODE REVIEW

File: src/containers/popup.jsx [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+// Reference:https://github.com/PaulLeCam/react-leaflet/issues/33
+
+const popupOptions = {
+  maxWidth: 'auto',
+  maxHeight: '300',
+  className: 'leaflet_popup',
+};
+
+const parseGeoJson = data =>
+  Object.entries(data).map(([key, value]) => {
```
Comment (joephayes) [inline src/containers/popup.jsx]: Suggestion on how to rewrite geojson mapping to handle arrays (prints keyname for first array entry and blank for the others):

```
const parseGeoJson = data =>
  Object.entries(data).map(([key, value]) => {
    if (value) {
      if (Array.isArray(value)) {
        return value.reduce((accumulator, element, currentIndex) => {
          const keyValue = currentIndex === 0 ? key : '&nbsp;';
          accumulator += `<tr><td>${keyValue}</td><td>${element}</td></tr>`; // eslint-disable-line no-param-reassign
          return accumulator;
        }, '');
      }
      return typeof value === 'object'
        ? parseGeoJson(value)
        : `<tr>
          <td>${key}</td>
          <td>${value}</td>
          </tr>`;
    }
    return `<tr>
            <td>${key}</td>
            <td>unknown</td>
            </tr>`;
  });

```

Comment (RyanMarshall5765) [reply src/containers/popup.jsx]: I like this. Implemented it and added formatting to the key strings.

Comment (RyanMarshall5765) [reply src/containers/popup.jsx]: Still trying to figure out where the commas are coming in from. There are 29 if that has any significance to you in correlation with the data.

File: src/containers/popup.jsx [hunk 2]
```diff
@@ -0,0 +1,55 @@
+// Using leaflet API for binding popup due to using <GeoJson />.
```
Comment (joephayes) [inline src/containers/popup.jsx]: Add some basic tests for the popup (see other test files for examples)

File: src/containers/popup.jsx [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+        : `<tr>
+          <td>${formatKey(key)}</td>
+          <td>${value}</td>
+          </tr>`;
+    }
+    return `<tr>
+            <td>${formatKey(key)}</td>
+            <td>unknown</td>
+            </tr>`;
+  });
```
Comment (joephayes) [inline src/containers/popup.jsx]: missing `join('')`?

**Pull request merged**